<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Navigation II</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVBoxLayout" name="vboxLayout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>471</width>
       <height>271</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pagedown">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PageDown</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pageup">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PageUp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="top">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>top</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bottom">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>bottom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>&amp;file</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionSearch"/>
    <addaction name="actionQuit"/>
    <addaction name="actionOpen"/>
    <addaction name="actionNewtab"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionMark_as_signature"/>
    <addaction name="action_file_format"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="actionScroll_up"/>
    <addaction name="actionScroll_down"/>
    <addaction name="actionPage_up"/>
    <addaction name="actionPage_down"/>
    <addaction name="navActionSearch"/>
   </widget>
   <addaction name="file"/>
   <addaction name="menuEdit"/>
   <addaction name="menuNavigation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>&amp;close</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;search</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;preferences</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>&amp;cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>cop&amp;y</string>
   </property>
  </action>
  <action name="actionMark_as_signature">
   <property name="text">
    <string>&amp;mark as signature</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;open</string>
   </property>
  </action>
  <action name="actionScroll_up">
   <property name="text">
    <string>&amp;scroll up </string>
   </property>
  </action>
  <action name="actionScroll_down">
   <property name="text">
    <string>s&amp;croll down</string>
   </property>
  </action>
  <action name="actionPage_up">
   <property name="text">
    <string>&amp;page up </string>
   </property>
  </action>
  <action name="actionPage_down">
   <property name="text">
    <string>page &amp;down</string>
   </property>
  </action>
  <action name="navActionSearch">
   <property name="text">
    <string>s&amp;earch</string>
   </property>
  </action>
  <action name="actionNewtab">
   <property name="text">
    <string>&amp;newtab</string>
   </property>
  </action>
  <action name="action_file_format">
   <property name="text">
    <string>&amp;file format</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVBoxLayout</class>
   <extends>QWidget</extends>
   <header>qvboxlayout.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
